### DISCUSSION TITLE
Some kind of navigation back to other work

### DISCUSSION THREAD

**Discussion by spydmobile** [community/community#10037] [Projects and Issues]
Currently when you are in this new project, there is no navigation to elsewhere except your user menu. It would be nice to have a link to other projects or repos, or anything.

**Comment by spydmobile** [comment]
This is referring to the Beta Projects

**Comment by berezovskyi** [comment]
Ideally, an org link for an org-wide project.

**Comment by VincentAudette** [comment]
The way I see this is to have links to all repositories that have added this project.

**Comment by asheroto** [comment]
👍 

Right now I have to use the top-right button, go back to Your Organizations, then Projects... just need a back button :-)

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
